analyzer-alert/src/main/java/cloud/erda/analyzer/alert/Main.java

```diff
@@ -191,18 +189,27 @@ public static void main(String[] args) throws Exception {
 
 
         // 存储告警记录
-        alertRender
-                .map(new AlertRecordMapFunction())
+        //不进行数据存储操作，将数据发送到kafka中，由monitor读取再存入mysql中
+        alertRender.
+                map(new AlertRecordMapFunction())
                 .setParallelism(parameterTool.getInt(STREAM_PARALLELISM_OPERATOR))
-                .addSink(new AlertRecordSink(parameterTool.getProperties()))
-                .setParallelism(parameterTool.getInt(STREAM_PARALLELISM_OUTPUT))
-                .name("store alert record");
-        // 存储通知记录
+                .name("RenderedAlertEvent to record")
+                .addSink(new FlinkKafkaProducer<>(
+                        parameterTool.getRequired(Constants.KAFKA_BROKERS),
+                        parameterTool.getRequired(Constants.TOPIC_RECORD_ALERT),
+                        new RecordSchema(AlertRecord.class)))
```

Comment:
There is already MetricEventSchema that can be used directly on the kafka sink, no need to reimplement RecordSchema

Reply:
MetricEventSchema can only store original alarm records.Should I change it to DeserializationSchema<T>, SerializationSchema<T>